Week2/muhannad.js

```diff
@@ -0,0 +1,144 @@
+let favoriteAnimals = ['rabbit', 'cow', 'dog', 'cat'];
+//6.5 Log your array.
+console.log(favoriteAnimals);
+//6.6 Add a statement that adds Daan's favorite animal ('baby pig') to the existing array.
+favoriteAnimals.push('baby pig');
+//6.7 Log your new array!
+console.log(favoriteAnimals);
+//7. More strings
+//Let's consider the following string: let myString = "this is a test".
+//7.1 Add the string to your file and console.log it.
+let myString = 'this is a test';
+console.log(myString);
+//7.2 Find a way to get the length of myString.
+//7.3 console.log the length of myString.
+console.log(myString.length);
+///////////////////////
+//8. Write a program that checks the types of two variables and prints out SAME TYPE if they are the same type.
+//8.1 First declare at least four variables and assign them different data types.
+let v1 = 'text1';
+let v2 = 'text2';
+let v3 = 25;
+let v4 = false;
+//8.2 For each variable write a console.log statement that logs the value
+console.log(v1, v2, v3, v4);
+//8.3 Now write a console.log statement wherein you first explain in words what you think the type of your variables is.
+console.log('v1:string , v2:string , v3:number , v4:boolean');
+//8.4 Now use typeof to log the actual type of your variables.
+let checkV1 = typeof v1;
+let checkV2 = typeof v2;
+let checkV3 = typeof v3;
+let checkV4 = typeof v4;
+console.log(checkV1, checkV2, checkV3, checkV4);
+//8.5 Now compare the types of your different variables with one another.
+if (checkV1 === checkV2) {
+  console.log('SAME THE TYPE');
+} else {
+  console.log('NOT THE SAME TYPE');
+}
+//8.6 Make sure to also show a message when the variables you are comparing are not the same type.
+if (checkV3 === checkV4) {
+  console.log('SAME THE TYPE');
+} else {
+  console.log('NOT THE SAME TYPE');
+}
+//9. If x equals 7, and the only other statement is x = x % 3, what would be the new value of x?
+//9.1 Add at least 3 console.log statements in which you show that you understand what % does.
+let result = x % 3;
+console.log(result);
+console.log(34 % 10); //    ==>   it give 4
+console.log(5 % 2); //    ==>   it give 1
+console.log(15 % 4); //    ==>   it give 3
+//10.1 Can you store multiple types in an array? Numbers and strings? Make an example that illustrates your answer.
+var information = [
+  'javascript',
+  15,
+  true,
+  (arr2 = ['firstIndex', 'secondIndex', 'thirdIndex']),
+  x,
+  'jquery',
+];
+//show latest Array:information
+for (let i = 0; i < information.length; i++) {
+  console.log(information[i]);
+}
+//10.2 Can you compare infinities? (Not in Eyad's world) - does 6/0 === 10/0? How can you test this?
+//10.3 Add console.log statements to the above program in which you show that you understand the concepts (just like you've done in the above assignments).
+let number1 = 6 / 0,
+  number2 = 10 / 0;
+console.log(number1, number2);
+console.log(number1.typeOf, number2.typeOf); //can not divide never by 0 because it give infinity
```

Comment:
Hi @Muhannad1989 , what are you trying to do here in line 127. Can you please elaborate?